app/src/main/java/org/oppia/android/app/testing/FractionInputInteractionViewTestActivity.kt

```diff
@@ -0,0 +1,44 @@
+package org.oppia.android.app.testing
+
+import android.view.View
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+import org.oppia.android.app.model.Interaction
+import org.oppia.android.app.model.WrittenTranslationContext
+import org.oppia.android.app.player.state.answerhandling.AnswerErrorCategory
+import org.oppia.android.app.player.state.answerhandling.InteractionAnswerErrorOrAvailabilityCheckReceiver
+import org.oppia.android.app.player.state.itemviewmodel.FractionInteractionViewModel
+import org.oppia.android.app.player.state.listener.StateKeyboardButtonListener
+import org.oppia.android.app.translation.AppLanguageResourceHandler
+import org.oppia.android.domain.translation.TranslationController
+import javax.inject.Inject
+
+class FractionInputInteractionViewTestActivity :
+  InjectableAppCompatActivity(),
+  StateKeyboardButtonListener,
+  InteractionAnswerErrorOrAvailabilityCheckReceiver {
+
+  @Inject
+  lateinit var resourceHandler: AppLanguageResourceHandler
```

Comment:
We had introduced inject function for this file in `ActivityComponentImp`, but we are not calling it. 

The way we call the inject function is 

```kt
(activityComponent as ActivityComponentImpl).inject(this)
```
without this, these `@inject` dependency injections will not work.